lib/coins.py

```diff
@@ -295,6 +297,16 @@ def electrum_header(cls, header, height):
             'nonce': nonce,
         }
 
+    @util.cachedproperty(expire_time=86400)
+    async def latest_version(cls):
+        if not hasattr(cls, "UPDATECHECK_URL"):
+            return None
+        async with aiohttp.ClientSession() as session:
+            async with session.get(cls.UPDATECHECK_URL) as response:
+                if response.status == 200:
+                    return await response.text()
+
+
 
```

Comment:
Can you change the variable name to LATEST_CLIENT_VERSION_URL and have the base class set to None, overriding in the Bitcoin class?

The cached property and aiohttp logic should be somewhere else; probably in controller.py as there is a single instance of that class.